code-examples/quarkus-kafka-quickstart/src/main/resources/application.properties

```diff
@@ -1,6 +1,9 @@
 # Quarkus config
 quarkus.ssl.native=true
 
+# Kafka bootstrap server
+kafka.bootstrap.servers=localhost:9092
```

Comment:
current config uses concept of profiles. 

When running quarkus:dev we use `dev.kafka.bootstrap.servers` etc.
We should not have those values without profiles - when profile is used.